test/parachain/src/command.rs

```diff
@@ -34,35 +34,36 @@ use sp_runtime::{
 use std::{net::SocketAddr, sync::Arc};
 
 impl SubstrateCli for Cli {
-	fn impl_name() -> &'static str {
-		"Cumulus Test Parachain Collator"
+	fn impl_name() -> String {
+		"Cumulus Test Parachain Collator".into()
 	}
 
-	fn impl_version() -> &'static str {
-		env!("SUBSTRATE_CLI_IMPL_VERSION")
+	fn impl_version() -> String {
+		env!("SUBSTRATE_CLI_IMPL_VERSION").into()
 	}
 
-	fn description() -> &'static str {
+	fn description() -> String {
 		"Cumulus test parachain collator\n\nThe command-line arguments provided first will be \
 		passed to the parachain node, while the arguments provided after -- will be passed \
 		to the relaychain node.\n\n\
 		cumulus-test-parachain-collator [parachain-args] -- [relaychain-args]"
+			.into()
 	}
```

Comment:
```suggestion
	fn description() -> String {
		format!("Cumulus test parachain collator\n\nThe command-line arguments provided first will be \
		passed to the parachain node, while the arguments provided after -- will be passed \
		to the relaychain node.\n\n\
		{} [parachain-args] -- [relaychain-args]", Self::executable_name())
	}
```

Not tested but I believe this should work